wp-includes/widgets/class-wp-widget-image.php

```diff
@@ -213,6 +215,27 @@ public function render_media( $instance ) {
 	}
 
 	/**
+	 * Filters the default media display states for items in the Media list table.
+	 *
+	 * @since 4.8.0
+	 * @access public
+	 *
+	 * @param array   $states An array of media states.
+	 * @param WP_Post $post   The current attachment object.
+	 * @return array
+	 */
+	public function display_media_state( $states, $post ) {
+		$settings      = $this->get_settings();
+		$attachment_id = empty( $settings[ $this->number ]['attachment_id'] ) ? 0 : $settings[ $this->number ]['attachment_id'];
```

Comment:
This is only going to work if you have one image widget. If you have multiple image widgets, then you'd have to iterate over all `$settings` to check each one to see if it has `attachment_id` that is the same as `$post->ID`. As it is right now, it's only looking at the current `$this->number` widget instance, which is probably going to be the last one registered.